http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/ClusterHelper.scala

```diff
@@ -137,3 +229,14 @@ class ClusterHelper(
   }
 
 }
+
+object ClusterHelper {
```

Comment:
is there any benefit wrapping these case classes in `ClusterHelper` object?

Reply:
Nothing functional but I find the code easier to read that way.